### PR TITLE
Feature free purchase

### PR DESCRIPTION
**Pull request by pedromtec**
#### What problem is this solving?

Same [here](https://github.com/vtex-apps/order-payment/pull/13)

#### How should this be manually tested?

Choose the account `pilateslovers`. 
[App Store](https://pedrocheckout--extensions.myvtex.com/vtex-facebook-pixel/p)

I'm getting some issues to test the flow using `checkoutio` account because I can't choose a free shipping.  
[Checkout IO](https://pedrocheckout--checkoutio.myvtex.com/Produto-gratis/p)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Jira story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Related to / Depends on
[PR13](https://github.com/vtex-apps/order-payment/pull/13)

### CODE REVIEW

**Review by lucasecdb — APPROVED** [review]

**Review by arturpimentel — APPROVED** [review]

File: messages/pt.json [hunk 1]
```diff
@@ -24,5 +24,6 @@
   "store/checkout-payment.reviewPurchaseLabel": "Revisar compra",
   "store/checkout-payment.billingAddressLabel": "Endereço de fatura",
   "store/checkout-payment.newAddressLabel": "+ Novo endereço",
-  "store/checkout-payment.cardFormTitle": "Contéiner para o formulário de cartão de crédito"
+  "store/checkout-payment.cardFormTitle": "Contéiner para o formulário de cartão de crédito",
+  "store/checkout-payment.freePurchase": "Tu compra es gratis"
```
Comment (lucasecdb) [inline messages/pt.json:28]:
```suggestion
  "store/checkout-payment.freePurchase": "Sua compra é gratuita"
```

**Review by lucasecdb — CHANGES_REQUESTED** [review]

File: messages/pt.json [hunk 2]
```diff
@@ -24,5 +24,6 @@
   "store/checkout-payment.reviewPurchaseLabel": "Revisar compra",
   "store/checkout-payment.billingAddressLabel": "Endereço de fatura",
   "store/checkout-payment.newAddressLabel": "+ Novo endereço",
-  "store/checkout-payment.cardFormTitle": "Contéiner para o formulário de cartão de crédito"
+  "store/checkout-payment.cardFormTitle": "Contéiner para o formulário de cartão de crédito",
```
Comment (wdsrocha) [inline messages/pt.json:27]:
```suggestion
  "store/checkout-payment.cardFormTitle": "Contêiner para o formulário de cartão de crédito",
```

File: messages/pt.json (hunk 1 above)
Comment (pedromtec) [reply messages/pt.json:28]: Problems with rebase :/

File: messages/pt.json (hunk 2 above)
Comment (pedromtec) [reply messages/pt.json:27]: Ok!

**Review by wdsrocha — CHANGES_REQUESTED** [review]
The branch seems to have few conflicts with master. Could you rebase please?

**Review by lucasecdb — APPROVED** [review]

File: react/Payment.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import React, { useState, useRef, useCallback } from 'react'
+import React, { useState, useRef, useCallback, useEffect} from 'react'
```
Comment (wdsrocha) [inline react/Payment.tsx:1]: weird that eslint didn't catch that 🤔
for me, it only catched when I ran prettier on this specific file. anyone knows why?

Anyway, if you feel like fixing it, ok, but not obligatory in my opinion

**Review by wdsrocha — APPROVED** [review]

**Pull request merged**